javascript-modal/main.js

```diff
@@ -0,0 +1,13 @@
+var $modal = document.querySelector('.modal');
+
+var $purpleButton =document.querySelector('.purple');
+
+var $pinkButton = document.querySelector('.pink')
+
+$purpleButton.addEventListener('click', function purpleButton(event){
+  $modal.style.display = 'block';
+});
+
+$pinkButton.addEventListener('click', function pinkButton(event){
+  $modal.style.display = 'none';
```

Comment:
Instead of assigning values directly to properties through the DOM, you should make a class that you can add and remove. DOing it this way, this becomes inline styling, which will completely destroy specificity if you needed this to interact with other CSS styling.